CHANGES.md

```diff
@@ -31,6 +31,7 @@ Change Log
 * Added `Matrix4.computeView`.
 * Added `CullingVolume.fromBoundingSphere`.
 * Added `debugShowShadowVolume` to `GroundPrimitive`.
+* Implemented support for [WEB3D_quantized_attributes](https://github.com/KhronosGroup/glTF/blob/master/extensions/Vendor/WEB3D_quantized_attributes/README.md). [#3241](https://github.com/AnalyticalGraphicsInc/cesium/issues/3241)
```

Comment:
Move this to a new 1.23 section dated `2016-07-01`.

Once your blog post is published, we'll also update this to include a link to it.  For example, here's one we wrote on binary glTF last summer: http://cesiumjs.org/2015/06/01/Binary-glTF/ (your blog post will, of course, have a lot more performance numbers).